app/javascript/components/widgets/widgets/forest-change/fires-alerts/selectors.js

```diff
@@ -24,12 +25,22 @@ export const getData = createSelector(
   [getAlerts, getDataset],
   (data, dataset) => {
     if (!data || isEmpty(data)) return null;
+    const dataMaxYear = maxBy(data, 'year').year || null;
+    const dataMaxWeek =
+      maxBy(data.filter(el => el.year === dataMaxYear), 'week').week || null;
     const groupedByYear = groupBy(data, 'year');
     const years = [];
-    const latestFullWeek = moment().subtract(2, 'w');
     const lastWeek = {
-      isoWeek: latestFullWeek.isoWeek(),
-      year: latestFullWeek.year()
+      isoWeek:
+        dataMaxWeek ||
+        moment()
+          .subtract(2, 'w')
+          .week(),
```

Comment:
don't we want isoWeek here like above removed code?

Reply:
No, should be isoWeek. Fixed.